editor/tests/org.fusesource.ide.camel.editor.tests.integration/src/main/java/org/fusesource/ide/camel/editor/integration/globalconfiguration/dataformat/wizards/NewDataFormatWizardIT.java

```diff
@@ -63,8 +63,7 @@
 	@Parameters(name = "{0} - {1}")
 	public static Collection<Object[]> data() {
 		List<String> supportedCamelVersions = CamelModelFactory.getSupportedCamelVersions();
-		// TODO: update when a new version is available
-		Assertions.assertThat(supportedCamelVersions).hasSize(2);
+		Assertions.assertThat(supportedCamelVersions.size()>0).isTrue();

```

Comment:
That will require to adapt the test with each new version added to our supported ones. (means this is an always recurring test failure with each new version added). Do we really want that?